be/src/exec/vectorized/aggregator.cpp

```diff
@@ -733,8 +743,39 @@ void Aggregator::_init_agg_hash_variant(HashVariantType& hash_variant) {
         }
         }
     }
+
+    int real_fixed_size = 0;
+    // this optimization don't need to be limited to multi-column group by.
+    // single column like float/double/decimal/largeint could also be applied to.
```

Comment:
AggHashMapWithOneNumberKey should faster than `AggHashMapWithSerializedKeyFixedSize`